flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/serde/RexNodeSerdeTest.java

```diff
@@ -92,121 +519,129 @@ public void testRexNodeSerde(RexNode rexNode) throws IOException {
-                        Sarg.of(
-                                false,
-                                ImmutableRangeSet.of(
-                                        Range.closed(
-                                                BigDecimal.valueOf(1), BigDecimal.valueOf(10)))),
-                        FACTORY.createSqlType(SqlTypeName.INTEGER)),
-                rexBuilder.makeSearchArgumentLiteral(
-                        Sarg.of(
-                                false,
-                                ImmutableRangeSet.of(
-                                        Range.range(
-                                                BigDecimal.valueOf(1),
-                                                BoundType.OPEN,
-                                                BigDecimal.valueOf(10),
-                                                BoundType.CLOSED))),
-                        FACTORY.createSqlType(SqlTypeName.INTEGER)),
-                rexBuilder.makeSearchArgumentLiteral(
-                        Sarg.of(
-                                false,
-                                TreeRangeSet.create(
-                                        Arrays.asList(
-                                                Range.closed(
-                                                        BigDecimal.valueOf(1),
-                                                        BigDecimal.valueOf(1)),
-                                                Range.closed(
-                                                        BigDecimal.valueOf(3),
-                                                        BigDecimal.valueOf(3)),
-                                                Range.closed(
-                                                        BigDecimal.valueOf(6),
-                                                        BigDecimal.valueOf(6))))),
-                        FACTORY.createSqlType(SqlTypeName.INTEGER)),
-                rexBuilder.makeInputRef(FACTORY.createSqlType(SqlTypeName.BIGINT), 0),
-                rexBuilder.makeCorrel(inputType, new CorrelationId("$cor1")),
-                rexBuilder.makeFieldAccess(
-                        rexBuilder.makeCorrel(inputType, new CorrelationId("$cor2")), "f2", true),
-                // cast($1 as smallint)
-                rexBuilder.makeCast(
-                        FACTORY.createSqlType(SqlTypeName.SMALLINT),
-                        rexBuilder.makeInputRef(FACTORY.createSqlType(SqlTypeName.INTEGER), 1)),
-                // $1 in (1, 3, 5)
-                rexBuilder.makeIn(
-                        rexBuilder.makeInputRef(FACTORY.createSqlType(SqlTypeName.INTEGER), 1),
-                        Arrays.asList(
-                                rexBuilder.makeExactLiteral(new BigDecimal(1)),
-                                rexBuilder.makeExactLiteral(new BigDecimal(3)),
-                                rexBuilder.makeExactLiteral(new BigDecimal(5)))),
+                //
```

Comment:
Why these comments?

Reply:
that was a mistake and got removed in following commits